moped-editor/src/views/projects/projectView/ProjectSummary.js

```diff
@@ -3,7 +3,6 @@ import { useParams } from "react-router-dom";
 
 import ProjectSummaryTable from "./ProjectSummaryTable";
 import ProjectSummaryMap from "./ProjectSummaryMap";
-import ProjectSummaryEditMap from "./ProjectSummaryEditMap";
```

Comment:
Removing the editing state management from the ProjectSummary page, since that functionality moved to the Map tab. And you'll see further below that the ProjectSummaryEditMap can be deleted. @sergiogcx that's fine, right?

Reply:
Totally fine, and thank you for keeping the code lean and clean! 💇  ✨